# Enable validateNoImpureFunctionsInRender by default to catch impure render functions
# Unable to configure via next.config reactCompiler option
# TODO: Remove when upstream supports this config option https://github.com/vercel/next.js/issues/92365

diff --git a/dist/index.js b/dist/index.js
index ac4a79e..5d05dd0 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -84526,7 +84526,7 @@ var EnvironmentConfigSchema = external_exports.object({
    * Validates that effects are not used to calculate derived data which could instead be computed
    * during render.
    */
-  validateNoDerivedComputationsInEffects: external_exports.boolean().default(false),
+  validateNoDerivedComputationsInEffects: external_exports.boolean().default(true),
   /**
    * Validates against creating JSX within a try block and recommends using an error boundary
    * instead.
@@ -84560,7 +84560,7 @@ var EnvironmentConfigSchema = external_exports.object({
   /**
    * Validate against impure functions called during render
    */
-  validateNoImpureFunctionsInRender: external_exports.boolean().default(false),
+  validateNoImpureFunctionsInRender: external_exports.boolean().default(true),
   /**
    * Validate against passing mutable functions to hooks
    */
